services/datamanager/builtin/capture_test.go

```diff
@@ -134,28 +160,22 @@ func TestDataCaptureEnabled(t *testing.T) {
 			// Update to new config and let it run for a bit.
 			err = dmsvc.Update(context.Background(), updatedConfig)
 			test.That(t, err, test.ShouldBeNil)
-			time.Sleep(captureTime)
-
-			// Check if initial config captured (or not) as we'd expect.
-			initialCaptureFiles := getAllFiles(initCaptureDir)
-			if !tc.initialServiceDisableStatus && !tc.initialCollectorDisableStatus {
-				// TODO: check contents
-				test.That(t, len(initialCaptureFiles), test.ShouldBeGreaterThan, 0)
-				test.That(t, initialCaptureFiles[0].Size(), test.ShouldBeGreaterThan, emptyFileBytesSize)
-			} else {
-				test.That(t, len(initialCaptureFiles), test.ShouldEqual, 0)
-			}
+			oldCaptureDirFiles := getAllFiles(initCaptureDir)
 
-			// Check if updated config captured (or not) as we'd expect.
-			updatedCaptureFiles := getAllFiles(updatedCaptureDir)
+			time.Sleep(captureTime)
 			if !tc.newServiceDisableStatus && !tc.newCollectorDisableStatus {
-				//TODO: check contents
+				updatedCaptureFiles := getCapturedFiles(updatedCaptureDir)
```

Comment:
[q]: Did the TODO check contents mean checking size and verifying file bytes or something else? Should we make tickets?

Reply:
Yup it meant exactly that. It can write some variable number of things, so I think we just want to check that  it wrote N > 1 sensor data, and that they matched what we expect. Added that